<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyOrderDetail</class>
 <widget class="QDialog" name="MyOrderDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Consolas&quot;;
</string>
   </property>
   <property name="text">
    <string>PRODUCT DETAILS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Consolas&quot;;
</string>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="address">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;
border-radius:  13px; 
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 10pt &quot;Consolas&quot;;
padding-left: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>180</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:  20pt &quot;Consolas&quot;;
</string>
   </property>
   <property name="text">
    <string>PAYMENT DETAILS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Consolas&quot;;
</string>
   </property>
   <property name="text">
    <string>Total Payment</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>678</x>
     <y>220</y>
     <width>111</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="m_price">
      <property name="styleSheet">
       <string notr="true">font: 15pt &quot;Consolas&quot;;
</string>
      </property>
      <property name="text">
       <string>$</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ship_price">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 15pt &quot;Consolas&quot;;
</string>
      </property>
      <property name="text">
       <string>$</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Consolas&quot;;
</string>
   </property>
   <property name="text">
    <string>ORDER DETAILS</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>231</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font: 15pt &quot;Consolas&quot;;
</string>
      </property>
      <property name="text">
       <string>Merchandise subtotal</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">font: 15pt &quot;Consolas&quot;;
</string>
      </property>
      <property name="text">
       <string>Shipping subtotal</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>371</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLabel" name="total_price">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>340</y>
     <width>129</width>
     <height>24</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Consolas&quot;;
</string>
   </property>
   <property name="text">
    <string>$</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
